Assets/MixedRealityToolkit/Interfaces/InputSystem/IMixedRealityPointer.cs

```diff
@@ -125,5 +125,10 @@ public interface IMixedRealityPointer : IEqualityComparer
         /// Called during the scene query just before the current pointer target changes.
         /// </summary>
         void OnPreCurrentPointerTargetChange();
+
+        /// <summary>
+        /// Resets pointer to initial state
```

Comment:
It would be good to add a "why" behind this so people understand the context -- we recycle pointers, not destroy them so when a hand is lost this should really reset the state completely.